watcher/src/watchers/AlgorandWatcher.ts

```diff
@@ -124,6 +127,6 @@ export class AlgorandWatcher extends Watcher {
     if (!vaasByBlock[toBlockKey]) {
       vaasByBlock[toBlockKey] = [];
     }
-    return vaasByBlock;
+    return { vaasByBlock, optionalBlockHeight: undefined };
```

Comment:
simply omit `optionalBlockHeight`

```suggestion
    return { vaasByBlock };
```

Reply:
Changed.

Reply:
Changed